spark/src/main/kotlin/com/adevinta/spark/components/buttons/Button.kt

```diff
@@ -230,6 +230,9 @@ public object SparkButtonDefaults {
         vertical = size.contentVerticalPadding,
     )
 
+    @Composable
+    internal fun Color.disabled(): Color = this.copy(alpha = SparkTheme.colors.dim3)
```

Comment:
Not a huge one but since there no benefit to have a function here it's "simpler" in the consumer side so instead of writing  `SparkTheme.colors.primary.disabled()` they would write `SparkTheme.colors.primary.disabled` but it's oka yif this is not done in this review as it's an internal utility function right now ^^

Reply:
Thanks for the detailed answer. I have updated the review ;-)